pennylane_lightning/src/gates/cpu_kernels/GateImplementationsAVXCommon.hpp

```diff
@@ -72,12 +82,20 @@ class GateImplementationsAVXCommon
         GateOperation::Rot,        GateOperation::CNOT,
         GateOperation::CZ,         GateOperation::SWAP,
         GateOperation::IsingXX,    GateOperation::IsingYY,
-        GateOperation::IsingZZ,
-        /* CY, IsingXY, ControlledPhaseShift, CRX, CRY, CRZ, CRot */
+        GateOperation::IsingZZ,    GateOperation::CY,
+        GateOperation::IsingXY,    GateOperation::ControlledPhaseShift,
+        GateOperation::CRY,        GateOperation::CRZ,
+        GateOperation::CRX,
+        /* CRot */
     };
 
     constexpr static std::array implemented_generators = {
-        GeneratorOperation::RX, GeneratorOperation::RY, GeneratorOperation::RZ};
+        GeneratorOperation::PhaseShift, GeneratorOperation::RX,
+        GeneratorOperation::RY,         GeneratorOperation::RZ,
+        GeneratorOperation::IsingXX,    GeneratorOperation::IsingYY,
+        GeneratorOperation::IsingZZ,
+        /* IsingXY, CRX, CRY, CRZ, ControllPhaseShift */
```

Comment:
Are we going to work on these gates later?

Reply:
Those are generators. I will go through them in the next PR.